lib/resource/belongs_to_actor.ex

```diff
@@ -0,0 +1,48 @@
+defmodule AshPaperTrail.Resource.BelongsToActor do
+  @moduledoc "Represents a belongs_to_actor relationship on a version resource"
+
+  defstruct [
+    :name,
+    :destination,
+    :define_attribute?,
+    :api
+  ]
+
+  @type t :: %__MODULE__{
+          name: atom,
+          destination: Ash.Resource.t(),
+          define_attribute?: boolean
+        }
```

Comment:
I look at Ash BelongsTo for inspiration. api was missing from that type. Should I have it?

Reply:
I'll create an issue. We can add a `actor_attribute` or something that can store a non-resource.